static/styles/left-sidebar.scss

```diff
@@ -193,6 +193,12 @@ li.active-sub-filter {
     margin-right: 15px;
 }
 
+#starred_messages_filter .count {
```

Comment:
I would just use `.top_left_starred messages .count` here, so we don't need to introduce extra markup.